Section 1.2: Short Answers

miaisakovic opened the pull request:
Developed the frontend for the Short Answers page in the Hacker form.

Later, the handleSubmit function in short-answers/page.tsx will need to be updated to direct the user to Section 1.3.

![image](https://github.com/user-attachments/assets/cb2376a4-ae4e-4c00-a547-971408b484f7)

geashley commented:
this is great! would you be able to add a verification of word count? like highlight the box in red if the word count is over 150 for the first question. and the word count limit for the second question is 200 words, could you add that in text and add a limit as well? thanks!

miaisakovic commented:
Thanks Ashley! I’ve updated the word limit for each question to 150, 200, 150, and 10 respectively. I also added validation to the handleChange function in short-answers/page.tsx. Users can no longer exceed the word limit (plus, if they try to paste text that goes over the limit, only the first allowed number of words will be retained).
